src/ontology/fbbt-edit.obo

```diff
@@ -146106,6 +146106,96 @@ relationship: develops_from FBbt:00059159 ! cyst cell of female germline 8-cell
+property_value: http://purl.org/dc/terms/contributor http://orcid.org/0000-0002-6095-8718
+property_value: http://purl.org/dc/terms/date 2022-09-09T12:08:12Z xsd:dateTime
+
+[Term]
+id: FBbt:00059162
+name: peripodial epithelial cell of leg disc
+def: "." [FBC:Autogenerated]
+subset: cur
+intersection_of: FBbt:00000124 ! epithelial cell
+intersection_of: part_of FBbt:00049581 ! peripodial epithelium of leg disc
+property_value: http://purl.org/dc/terms/contributor http://orcid.org/0000-0002-6095-8718
+property_value: http://purl.org/dc/terms/date 2022-09-09T12:10:59Z xsd:dateTime
+
+[Term]
+id: FBbt:00059163
+name: epithelial cell of leg disc
+def: "." [FBC:Autogenerated]
+subset: cur
+intersection_of: FBbt:00000124 ! epithelial cell
+intersection_of: part_of FBbt:00059161 ! leg disc epithelium proper
+property_value: http://purl.org/dc/terms/contributor http://orcid.org/0000-0002-6095-8718
+property_value: http://purl.org/dc/terms/date 2022-09-09T12:12:57Z xsd:dateTime
+
+[Term]
+id: FBbt:00059164
+name: distal epithelial cell of leg disc
+def: "Epithelial cell of the leg disc that gives rise to the claw and the tarsal segments of the leg. It is located in the central part of the leg disc. It expresses Distal-less (Estella et al., 2012; Tse et al., 2022)." [FlyBase:FBrf0217387, FlyBase:FBrf0253840]
+subset: cur
+is_a: FBbt:00059163 ! epithelial cell of leg disc
+relationship: expresses http://flybase.org/reports/FBgn0000157 ! Dll
+property_value: http://purl.org/dc/terms/contributor http://orcid.org/0000-0002-6095-8718
+property_value: http://purl.org/dc/terms/date 2022-09-09T12:59:07Z xsd:dateTime
+
+[Term]
+id: FBbt:00059165
+name: medial epithelial cell of leg disc
+def: "Epithelial cell of the leg disc that gives rise to the femoral and tibial segments of the leg. It is located in a ring around the central-most part of the leg disc. It expresses dachshund (Estella et al., 2012; Tse et al., 2022)." [FlyBase:FBrf0217387, FlyBase:FBrf0253840]
+subset: cur
+is_a: FBbt:00059163 ! epithelial cell of leg disc
+relationship: expresses http://flybase.org/reports/FBgn0005677 ! dac
+property_value: http://purl.org/dc/terms/contributor http://orcid.org/0000-0002-6095-8718
+property_value: http://purl.org/dc/terms/date 2022-09-09T13:07:19Z xsd:dateTime
+
+[Term]
+id: FBbt:00059166
+name: proximal epithelial cell of leg disc
+def: "Epithelial cell of the leg disc that gives rise to the coxa and the trochanter. It is located at the periphery of the leg disc. It expresses homothorax (Estella et al., 2012; Tse et al., 2022)." [FlyBase:FBrf0217387, FlyBase:FBrf0253840]
+subset: cur
+is_a: FBbt:00059163 ! epithelial cell of leg disc
+relationship: expresses http://flybase.org/reports/FBgn0001235 ! hth
+property_value: http://purl.org/dc/terms/contributor http://orcid.org/0000-0002-6095-8718
+property_value: http://purl.org/dc/terms/date 2022-09-09T13:09:48Z xsd:dateTime
+
+[Term]
+id: FBbt:00059167
+name: neuron of leg disc
```

Comment:
Beira and Paro (FBrf0233457) (for all discs) and Tripathi and Irvine (FBrf0253125) (for wing discs specifically, not leg discs) explicitly mention the presence of _neurons_ (among other non-epithelial cells) in discs, but it may very well an abusive use of the term to refer more broadly to any cell of a neuronal lineage.

In the 3rd instar larval wing disc, Zappia et al. (FBrf0247435) reported only _sensory organ precursor cells_ that are “undergoing cytodifferentiation into neuronal receptors”.

So I agree on the safest action for now being to refer to these cells as neuronal precursors instead of fully differentiated neurons, until and unless more is known about the timing of their differentiation (e.g., is it completely over before the end of the 3rd instar larval stage?).